packages/core/src/components/Button/styles/outlinedButton.tsx

```diff
@@ -17,8 +17,18 @@ const getStyle = (textColor: string, bgColor = 'transparent', borderColor: strin
     }
 `;
 
+const getHoverShadow = (hoverShadow: boolean, shadowColor: string ) => {
```

Comment:
You can write this like below
```ts
const getHoverShadow = (shadowColor: string ) => css`
      box-shadow: 0 0.2rem 0.8rem ${rgba(shadowColor, 0.35)};
    `
```

and then you can use is like 
```ts
${hoverShadow && getHoverShadow(borderColor?.hovered || textColor.hovered)}
```

Reply:
Thanks, that's much cleaner, I have implemented it that way.